rest_framework/locale/ach/LC_MESSAGES/django.po

```diff
@@ -1,7 +1,7 @@
 # SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the PACKAGE package.
-# 
+#
```

Comment:
I'm not familiar with transifex so I might be wrong here, but aren't the .po files regenerated by transifex for every release? It seems like the whitespace changes there would be superfluous.

Reply:
@rpkilby Yes. That's we all the `po` files and the `compilemessages` from those. Good catch.